diff --git a/node_modules/@jest/fake-timers/build/modernFakeTimers.js b/node_modules/@jest/fake-timers/build/modernFakeTimers.js
index 976ed53..3283f54 100644
--- a/node_modules/@jest/fake-timers/build/modernFakeTimers.js
+++ b/node_modules/@jest/fake-timers/build/modernFakeTimers.js
@@ -119,7 +119,7 @@ class FakeTimers {
 
   useFakeTimers() {
     if (!this._fakingTime) {
-      const toFake = Object.keys(this._fakeTimers.timers);
+      const toFake = Object.keys(this._fakeTimers.timers).filter(key => key !== "nextTick" && key !== "queueMicrotask");
       this._clock = this._fakeTimers.install({
         loopLimit: this._maxLoops,
         now: Date.now(),
